src/avram/migrator/columns/add_string.cr

```diff
@@ -0,0 +1,14 @@
+require "./base"
+
+module Avram::Migrator::Columns
+  class AddString < Base
+    @default : String? = nil
+
+    def initialize(@name, @nilable, @default)
+    end
+
+    def column_type
+      "text"
+    end
+  end
+end
```

Comment:
Should this class support a length option?

Reply:
It could, but I think I'd rather keep this PR purely refactoring and not  adding new stuff. I'm also not sure it should be added since `text` columns can be any length. But I dunno. Probably wouldn't be too bad to  add later

Reply:
True. I definitely think it should be added, but we can go through all that stuff and start adding some fancy add-ons later.

Reply:
That sounds fair to me! Luckily this refactor should make this stuff a lot easier to do! I'm sure we could add some time options and stuff too